requirements.txt

```diff
@@ -1,15 +1,20 @@
 # Requirements
 Flask>=0.12
 Flask-SQLAlchemy>=2.2
-SQLAlchemy>=1.1.7
+SQLAlchemy
 pytz>=2017.2
 psycopg2-binary>=2.8.1
+alembic>=1.5.1
+six>=1.15.0
 
 # Testing & documentation requirements (note these must be installable across all Tox environment)
-pytest==3.0.7
+pytest>=3.0.7
 codecov
 pytest-cov==2.4.0
 Flask-Security-Fork==2.0.1
 sphinx==1.5.5
 sphinx-autobuild==0.6.0
 Flask-Sphinx-Themes==1.0.1
+bcrypt>=3.0.0
+email_validator>=1.1.0
+enum34>=1.1.10
```

Comment:
why do we need that?

Reply:
☝️  these are just for local dev and tests, right?